src/scripts/json-presets-to-new-format.ts

```diff
@@ -0,0 +1,30 @@
+import { DatabaseContext, getNewConnection } from "../database_context";
+
+async function exportJsonPresetsToNewTable(db: DatabaseContext): Promise<void> {
+    const jsonPresets = await db.kmq("game_option_presets_json").select("*");
+    await Promise.all(jsonPresets.map(async (preset) => {
+        const guildID = preset["guild_id"];
+        const presetName = preset["preset_name"];
+        const options = JSON.parse(preset["game_options"]);
+        await Promise.all(
```

Comment:
Let's add error handling here incase one of these fails and we need to do it manually